src/azure-cli/azure/cli/command_modules/ams/tests/latest/recordings/test_ams_streaming_endpoint.yaml

```diff
@@ -201,15 +201,15 @@ interactions:
           AZURECLI/2.0.43]
       accept-language: [en-US]
     method: DELETE
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001?api-version=2019-07-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest.rg000001?api-version=2018-05-01
```

Comment:
It looks there is something wrong here. The api version for resource group should be 2019-07-01. Why can you use 2018-05-01 here?

Reply:
Removed the test results for tests that no longer exist.